components/TARGET_PSA/services/attestation/qcbor/test/qcbor_encode_tests.c

```diff
@@ -0,0 +1,2007 @@
+/*==============================================================================
+ Copyright (c) 2016-2018, The Linux Foundation.
+ Copyright (c) 2018-2019, Laurence Lundblade.
+ All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
```

Comment:
Any non apache files should be documented . See https://os.mbed.com/docs/mbed-os/v5.11/contributing/license.html

This needs addition to our Readme.md and LICENSE.md file with license description in the attestation folder or whichever folder its root for this software (I can see Readme there, that is good ! Just add LICENSE.md with the license and add reference in our root Readme).

Reply:
Done.